src/messaging/classes/Auth.ts

```diff
@@ -128,11 +128,15 @@ export class Auth {
             throw new AuthError(Constants.ERRORS.AUTH_ERROR);
         }
 
-        const credentialsFromStorage = await getConfig();
+        const credentialsFromStorage = await getAuthConfig();
```

Comment:
was wondering if the storage error could be propagated from this call to `getAuthConfig`....is that a problem?

Reply:
How about something like this?

`
        const credentialsFromStorage = await getAuthConfig() as TAuthConfig;
        if (!credentialsFromStorage) {
            throw new AuthError(Constants.ERRORS.AUTH_ERROR);
        }
`

where getAuthConfig() will not throw StorageError but could return TAuthConfig | boolean?

`
export const getAuthConfig = async (): Promise<TAuthConfig | boolean> => {
    try {
        const data = await storage().get({
            bucket: Constants.BUCKET_AUTH_STORE,
            key: Constants.BUCKET_KEY_CLIENT_CREDENTIALS,
        });
        return JSON.parse(data);
    } catch (error) {
        return false;
    }
};
`

Reply:
yeah that sounds good...or i was even thinking just use `.then().catch()` on the promise